app/templates/partials/question.html

```diff
@@ -30,7 +30,7 @@ <h3 class="question__subtitle mars">{{question_subtitle}}</h3>
       {{question.description}}
     {% for content in question.content %}
         <div class="question__description mars">
-            {{content.description}}
+            <p>{{content.description}}</p>
```

Comment:
That's not what I meant - that's still rendering a div for every line of the description:

Currently it's this:
```
<div class="question__description mars">
      {{question.description}}
    {% for content in question.content %}
        <div class="question__description mars">
            <p>{{content.description}}</p>
        </div>
    {% endfor %}
    </div>
```

But I think it'd be better as this:
```
<div class="question__description mars">
  {{question.description}}
  {% for content in question.content %}
    <p>{{content.description}}</p>
  {% endfor %}
</div>
```